tests/appsec/test_ip_blocking.py

```diff
@@ -54,8 +54,8 @@ def validate(data):
         interfaces.library.validate_remote_configuration(validator=validate)
 
     def setup_blocked_ips(self):
```

Comment:
I would set a different test methods for both the range and the not blocked ip, with explicit comments about why is it supposed to (or not to) block.

I bet lot of beer it will help during implementation